src/status_im2/contexts/chat/home/view.cljs

```diff
@@ -27,7 +27,8 @@
          (filter k)
          (sort-by :timestamp >))))
 
-(def empty-state-content
+(defn empty-state-content
```

Comment:
Apologies, my bad, I'm using `theme/theme-value` in a `def` instead of a `defn` :man_facepalming: